packages/vitaminjs/package.json

```diff
@@ -4,7 +4,8 @@
   "description": "The build toolchain against JavaScript Fatigue",
   "main": "index.js",
   "bin": {
-    "vitamin": "./bin/vitamin.js"
+    "vitamin": "./bin/vitamin.sh",
+    "vitamin-version": "./bin/vitamin-version.js"
```

Comment:
does it needs to be added in this PR?

Reply:
This is the result of a bad merge. I'm fixing it